docs/modules/ROOT/pages/system-properties.adoc

```diff
@@ -569,6 +569,13 @@ idle CPU usage but may result in decreased latency. Higher values will increase
 |int
 |Some JMX MBeans are cached  to reduce the overhead of calculating statistics. This parameter determines for how long the MBeans can go stale.
 
+|`hazelcast.journal.cleanup.threshold`
+| 1.0
+| float
+| The cleanup threshold for the event journal, expressed as a fraction of its capacity (a value between 0.0 and 1.0).
+When the remaining capacity falls below this threshold, a cleanup may be triggered during put operations to remove expired events (only if a xref:data-structures:event-journal.adoc#configuring-event-journal-ttl[time-to-live] for event journal is configured).
```

Comment:
> during put operations

I am wondering (maybe nit-picking) if it is clear that we mean here journal/ringbuffer put operation and not IMap/ICache put operation